README.rst

```diff
@@ -110,21 +214,41 @@ Inform getpaid of your Order model in ``settings.py`` and provide settings for p
             "second_key": "91ae651578c5b5aa93f2d38a9be8ce11",
             "oauth_id": 12345,
             "oauth_secret": "12f071174cb7eb79d4aac5bc2f07563f",
+            "continue_url": PAYMENT_CONTINUE_URL,
+            "retry_url": PAYMENT_RETRY_URL,
         },
     }
 
+
 Write a view that will create the Payment.
 
 An example view and its hookup to urls.py can look like this:
 
 .. code-block:: python
 
-    # main urls.py
+    # orders/views.py
+    from rest_framework import mixins, permissions, viewsets
+    from getpaid.rest_framework.payment_creator import PaymentCreator
 
-    urlpatterns = [
-        # ...
-        path("gp/", include("getpaid.urls")),
-    ]
+    class OrderViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
+        serializer_class = OrderSerializer
+        queryset = Order.objects.all()
+        permission_classes = (permissions.IsAuthenticated,)
+
+        @transaction.atomic()
+        def perform_create(self, serializer):
+            super().perform_create(serializer)
+            self.create_payment(serializer.instance)
```

Comment:
Could you please add some info where the payment_url is stored and when it's available to pass to a user?

Reply:
It was unused code - removed.